domain/consensus/ruleerrors/rule_error.go

```diff
@@ -185,6 +185,9 @@ var (
 
 	//ErrTransactionVersionIsUnknown indicates that the transaction version is unknown.
 	ErrTransactionVersionIsUnknown = newRuleError("ErrTransactionVersionIsUnknown")
+
+	// ErrPrunedBlock indicates that the currently validated block is already pruned.
```

Comment:
The phrasing here could be misleading. Please write something like "ErrPrunedBlock indicates that the block currently being validated had already been pruned."